src/functions-and-arrays.js

```diff
@@ -52,14 +118,35 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  let result=""
+  if (!wordsUnique.length){
+    return null;
+  }
+  for(let i=0;i<wordsUnique.length;i++) {
+    if(array[i] === array[i+1]) {
+      return result= array.join[i]; 
+    }
+else 
+{
+  return  result=array[i];
+}
+  }
+  return result;
+}
+
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(wordsFind) {
```

Comment:
So we know that this `function` will accept two `arguments` the first one will be the `array` of words, and the other one will be the specific `word` that you want to check if it exists in the array. So what you will need to do is, create a `loop` that will run over the `array` and in which `element` it will check if `word` is equal to the current `element` of the array:

```
function doesWordExist(array, word) {
  if (array.length === 0) {
    return null;
  }
  for (let i = 0; i < array.length; i++) {
    if (array[i] === word) {
      return true;
    }
  }
  return false;
}
```